src/Lti/Message/DeepLinking/ScoreConstraints.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cerpus\EdlibResourceKit\Lti\Message\DeepLinking;
+
+final readonly class ScoreConstraints
```

Comment:
Message objects should be extendable and not readonly. It may be desirable to add additional stuff, as is done with LtiLinkItem.